src/cpp/session/modules/SessionCodeSearch.cpp

```diff
@@ -1399,13 +1401,16 @@ class SourceItem
 public:
    enum Type
    {
-      None = 0,
-      Function = 1,
-      Method = 2,
-      Class = 3,
-      Enum = 4,
+      None      = 0,
+      Function  = 1,
+      Method    = 2,
+      Class     = 3,
+      Enum      = 4,
       EnumValue = 5,
-      Namespace = 6
+      Namespace = 6,
+      Section   = 7,
```

Comment:
Do we want to distinguish between heading levels here? (Hadley had asked for distinguishing headings levels in xref completions b/c it's helpful context). Paul could do separate icons for H1-H6.

Reply:
Even just h1 vs h2 vs everything else would be useful